src/network/networkd-route.c

```diff
@@ -87,20 +87,68 @@ static const char * const route_table_table[] = {
         [RT_TABLE_LOCAL]   = "local",
 };
 
-DEFINE_STRING_TABLE_LOOKUP(route_table, int);
+DEFINE_PRIVATE_STRING_TABLE_LOOKUP(route_table, int);
 
-#define ROUTE_TABLE_STR_MAX CONST_MAX(DECIMAL_STR_MAX(int), STRLEN("default") + 1)
-static const char *format_route_table(int table, char *buf, size_t size) {
+int route_table_from_string_full(const Manager *m, const char *s, uint32_t *ret) {
```

Comment:
i'd name these manager_get_route_table_by_string() btw...

the xyz_from_abc() and xyz_to_abc() functions are mostly stateless "pure" functions, i.e. that operate on completely static fixed tables. i.e. not dynamically registered stuff like this. 

but doesn't matter, this is just bikeshedding